src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -173,6 +180,9 @@ public DomainFormPanel setInferFieldFile(File file)
         return this;
     }
 
+    // TODO: add ability to select "import data"/"don't import" here, after inferring fields from file
+    // for datasets, the ability to map key columns is exposed when 'import data' is selected
```

Comment:
Where are the helpers for that "Import Data" toggle for the list designer case? Or did the list tests not need to implement that at this time?

Reply:
I'm positive that I implemented support for that slider.  I don't know where to find it, tho.